### ISSUE TITLE
Add an appropriate <title> for example page template

### ISSUE THREAD

**Issue by yatesn** [ONSdigital/design-system#1932]
Currently hardocoded to "example". Should include the component/pattern name.

This type of page https://ons-design-system.netlify.app/components/accordion/examples/accordion/

**Comment by jrbarnes9** [comment]
@yatesn Raised on page 54-55 of June 2022 DAC DS report that this causes duplicate page titles which fails WCAG Level A.

**Comment by jrbarnes9** [comment]
Fixed in #2380
